docs/kuberay/ray-service.aviary-eks.yaml

```diff
@@ -7,13 +7,23 @@ spec:
   deploymentUnhealthySecondThreshold: 1200 # Config for the health check threshold for deployments. Default value is 60.
   serveConfigV2: |
```

Comment:
this file needs to be left as is, the new format doesn't use multiple applications